manifests/d/dxatlas/CwSkimmer/2.1/dxatlas.CwSkimmer.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: dxatlas.CwSkimmer
+PackageVersion: 2.1
+PackageLocale: en-US
+Publisher: dxatlas
+PackageName: CwSkimmer
+License: none
+ShortDescription: Multi-channel CW decoder and analyzer.

```

Comment:
We can probably change the description to something like:

```suggestion
ShortDescription: Multi-channel Morse code (CW) decoder and analyzer.

```

Reply:
How do i change it

Reply:
Yeah, that's a little more clear 😃

Reply:
It should be possible to directly merge this change I sent. If not, you will have to make a new commit in your forked repo.

Reply:
Need i click resolve

Reply:
Or no, my bad. Clicking resolve will probably just close this thread. There should be another button.

Reply:
I open github in the browser i am now on mobile